posix-mapper/docker/config/posix-mapper/catalina.properties

```diff
@@ -0,0 +1,13 @@
+tomcat.connector.scheme=https
```

Comment:
accidental add/commit? this file should not be in version control

Reply:
Yup.  Removed.